### PR TITLE
Upgrade to Chromium 81.0.4044.92

### PR DESCRIPTION
**Pull request by Zoraver**
I will let you know once I have a successful build. I expect that I will need to do a few revisions to this patchset to get everything working.

### CODE REVIEW

**Comment by jstkdng** [comment]
building right now as well, hope all goes well
https://build.opensuse.org/package/live_build_log/home:justkidding:arch/ungoogled-chromium/standard/x86_64

**Comment by jstkdng** [comment]
Okay, just finished compiling and I'm getting a sig trap when trying to get to chrome://settings/help
```
[87783:87783:0409/103415.369753:ERROR:browser_switcher_service.cc(238)] XXX Init()
[87807:87807:0409/103415.748922:ERROR:sandbox_linux.cc(374)] InitializeSandbox() called with multiple threads in process gpu-process.
[1]    87783 trace trap (core dumped)  chromium
```
Will continue to test

**Comment by Zoraver** [comment]
I ran into the same issue as @jstkdng; loading chrome://settings or any of its subpages will crash the browser. I'm working on figuring out the source of the crash.

**Comment by Zoraver** [comment]
I have compiled Chromium 81.0.4044.92 with this patchset and it is running well. It no longer crashes when chrome://settings is opened, and I have not observed any other regressions. Feel free to merge.

**Comment by seanking2919** [comment]
Simply curious: With this version, are you able to add extensions to Ungoogled Chromium?

**Comment by braewoods** [comment]
It's been possible all along @seanking2919. It's just that they can't be installed from the store afaik. They have to be loaded locally.

**Comment by braewoods** [comment]
@Marek33 I keep seeing you thumbs up most of the comments on various issues or pull requests. Are you a bot or something? o.O

**Comment by seanking2919** [comment]
@braewoods I'm referring to this issue that we've been trying to resolve: https://github.com/Eloston/ungoogled-chromium/issues/923

**Comment by Marek33** [comment]
@braewoods No, but I'm glad when there are updates for a great software. Especially, since the Ungoogled, had one time an old version for a very long time. Plus, I'm used to giving people thumbs up, for good work, and comments.

**Comment by jstkdng** [comment]
@Zoraver 
Just tested this as well and all is working. Thanks for the work! Tried to help with the debugging but had problems with the OBS.

@seanking2919 
Only windows has problems with installing extensions. Under linux I'm able to install extensions without any problem.

**Comment by Eloston** [comment]
@seanking2919 I don't see any significant enough Safe Browsing changes that would fix drag-and-drop, but I'm not 100% certain. Since this issue has persisted for several major versions now, I'd assume the problem lies in a change that's been around for a while.

File: patches/core/ungoogled-chromium/block-trk-and-subdomains.patch [hunk 1]
```diff
@@ -33,35 +33,6 @@
...
-        it != protocol_handlers->end();
-        ++it) {
--    bool set_protocol =
-+    set_protocol =
-         job_factory->SetProtocolHandler(it->first, std::move(it->second));
-     DCHECK(set_protocol);
-   }
-+  set_protocol = job_factory->SetProtocolHandler(
-+      url::kTraceScheme, base::WrapUnique(new net::TrkProtocolHandler));
-+  DCHECK(set_protocol);
```
[the remark this answers is not in the document]
Comment (Zoraver) [reply patches/core/ungoogled-chromium/block-trk-and-subdomains.patch:61]: As far as I can tell, *chromecast/browser/url_request_context_factory.cc* was [entirely removed](https://github.com/chromium/chromium/commit/49a73db55ba2ce9991be181d2fffbaa0c820eefb) (not renamed), and the only place in the code where `SetProtocolHandler()` is called (outside of fuzzers/tests and iOS-specific code) is in *net/url_request/url_request_context_builder.cc* which is patched in the same manner that *url_request_context_factory.cc* was. I could have missed something though; it is a very large code base...

**Comment by PF4Public** [comment]
Just compiled on Gentoo with these patches. Looks functioning.

File: patches/core/ungoogled-chromium/block-trk-and-subdomains.patch (hunk 1 above)
[the remark this answers is not in the document]
Comment (Eloston) [reply patches/core/ungoogled-chromium/block-trk-and-subdomains.patch:61]: Oops yeah you're right; that's fine. Thanks for explaining!

**Comment by PF4Public** [comment]
New release doesn't appear to be created or am I missing something?

**Comment by Eloston** [comment]
@PF4Public Because I wanted to squash a bug in 2171d8ec9977f5fcf3b52df3df3a8fe91fcce3f1. New release is out.

**Comment by jstkdng** [comment]
@Eloston 
Just tried the new release but I'm getting this errors:
```
[  263s] clang++ -MMD -MF obj/components/ungoogled/switches/ungoogled_switches.o.d -DIS_UNGOOGLED_IMPL -DUSE_UDEV -DUSE_AURA=1 -DUSE_GLIB=1 -DUSE_NSS_CERTS=1 -DUSE_X11=1 -DOFFICIAL_BUILD -D_FILE_OFFSET_BITS=64 -D_LARGEFILE_SOURCE -D_LARGEFILE64_SOURCE -DNO_UNWIND_TABLES -D_GNU_SOURCE -DCR_CLANG_REVISION=\"n340759-eaabaf7e-1\" -D__STDC_CONSTANT_MACROS -D__STDC_FORMAT_MACROS -D_FORTIFY_SOURCE=2 -DNDEBUG -DNVALGRIND -DDYNAMIC_ANNOTATIONS_ENABLED=0 -DGLIB_VERSION_MAX_ALLOWED=GLIB_VERSION_2_40 -DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_40 -I../.. -Igen -I../../third_party/boringssl/src/include -fprofile-sample-use=../../chrome/android/profiles/afdo.prof -fno-strict-aliasing --param=ssp-buffer-size=4 -fstack-protector -fno-unwind-tables -fno-asynchronous-unwind-tables -fPIC -pthread -fcolor-diagnostics -fmerge-all-constants -fcrash-diagnostics-dir=../../tools/clang/crashreports -Xclang -mllvm -Xclang -instcombine-lower-dbg-declare=0 -flto=thin -fsplit-lto-unit -fwhole-program-vtables -m64 -march=x86-64 -no-canonical-prefixes -Wall -Wextra -Wimplicit-fallthrough -Wthread-safety -Wextra-semi -Wno-missing-field-initializers -Wno-unused-parameter -Wno-c++11-narrowing -Wno-unneeded-internal-declaration -Wno-undefined-var-template -Wno-ignored-pragma-optimize -Wno-implicit-int-float-conversion -Wno-final-dtor-non-final-class -Wno-bitwise-conditional-parentheses -Wno-builtin-assume-aligned-alignment -Wno-deprecated-copy -Wno-misleading-indentation -Wno-range-loop-analysis -O2 -fno-ident -fdata-sections -ffunction-sections -fno-omit-frame-pointer -g0 -fsanitize=cfi-vcall -fsanitize-blacklist=../../tools/cfi/blacklist.txt -fsanitize=cfi-icall -fvisibility=hidden -Wheader-hygiene -Wstring-conversion -Wtautological-overlap-compare -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -std=c++14 -fno-exceptions -fno-rtti -fvisibility-inlines-hidden -D_FORTIFY_SOURCE=2 -D__DATE__=  -D__TIME__=  -D__TIMESTAMP__= -march=x86-64 -mtune=generic -O2 -pipe -fno-plt -Wno-builtin-macro-redefined -Wno-unknown-warning-option -c ../../components/ungoogled/ungoogled_switches.cc -o obj/components/ungoogled/switches/ungoogled_switches.o
[  263s] In file included from ../../components/ungoogled/ungoogled_switches.cc:5:
[  263s] ../../components/ungoogled/ungoogled_switches.h:14:40: error: definition of variable with array type needs an explicit size or an initializer
[  263s] const char COMPONENT_EXPORT(UNGOOGLED) kFingerprintingClientRectsNoise[];
[  263s]                                        ^
[  263s] ../../components/ungoogled/ungoogled_switches.h:15:40: error: definition of variable with array type needs an explicit size or an initializer
[  263s] const char COMPONENT_EXPORT(UNGOOGLED) kFingerprintingCanvasMeasureTextNoise[];
[  263s]                                        ^
[  263s] ../../components/ungoogled/ungoogled_switches.h:16:40: error: definition of variable with array type needs an explicit size or an initializer
[  263s] const char COMPONENT_EXPORT(UNGOOGLED) kFingerprintingCanvasImageDataNoise[];
[  263s]                                        ^
[  263s] ../../components/ungoogled/ungoogled_switches.cc:10:12: error: redefinition of 'kFingerprintingClientRectsNoise'
[  263s] const char kFingerprintingClientRectsNoise[] = "fingerprinting-client-rects-noise";
[  263s]            ^
[  263s] ../../components/ungoogled/ungoogled_switches.h:14:40: note: previous definition is here
[  263s] const char COMPONENT_EXPORT(UNGOOGLED) kFingerprintingClientRectsNoise[];
[  263s]                                        ^
[  263s] ../../components/ungoogled/ungoogled_switches.cc:13:12: error: redefinition of 'kFingerprintingCanvasMeasureTextNoise'
[  263s] const char kFingerprintingCanvasMeasureTextNoise[] = "fingerprinting-canvas-measuretext-noise";
[  263s]            ^
[  263s] ../../components/ungoogled/ungoogled_switches.h:15:40: note: previous definition is here
[  263s] const char COMPONENT_EXPORT(UNGOOGLED) kFingerprintingCanvasMeasureTextNoise[];
[  263s]                                        ^
[  263s] ../../components/ungoogled/ungoogled_switches.cc:16:12: error: redefinition of 'kFingerprintingCanvasImageDataNoise'
[  263s] const char kFingerprintingCanvasImageDataNoise[] = "fingerprinting-canvas-image-data-noise";
[  263s]            ^
[  263s] ../../components/ungoogled/ungoogled_switches.h:16:40: note: previous definition is here
[  263s] const char COMPONENT_EXPORT(UNGOOGLED) kFingerprintingCanvasImageDataNoise[];
```

**Pull request merged**